flash_examples/finetuning/translation.py

```diff
@@ -0,0 +1,31 @@
+import flash
+from flash import download_data
+from flash.text import TranslationData, TranslationTask
+
+if __name__ == "__main__":
```

Comment:
I am not much fun of this script when all body is hidden in if/main

Reply:
this is primarily done for multiprocessing reasons, i think mp proc has issues when you do not include an if/main. If anything this is the format that the other scripts have been using!